simple/imm_data.c

```diff
@@ -105,20 +121,20 @@ static int alloc_ep_res(struct fi_info *fi)
 	cq_attr.size = rx_depth;
 	ret = fi_cq_open(dom, &cq_attr, &rcq, NULL);
 	if (ret) {
-		printf("fi_cq_open() send comp %s\n", fi_strerror(-ret));
+		FI_PRINTERR("fi_cq_open: recv completion", ret);
```

Comment:
We need to have the "call" argument since __func__ will only print the function where FI_PRINTERR is called say alloc_ep_res but in order to get the error for the particular fi_* call, we need to pass the fi function name.

I can change it in this PR. Let me know if I should create a new PR for this. @shefty